functionnal_tests/cypress/integration/notification_wall/mark_as_read_spec.js

```diff
@@ -18,18 +18,17 @@ describe('Notification Wall', () => {
     cy.cancelXHR()
   })
 
-  it('should mark the notification as read on click', () => {
-    cy.get('.notification__list__item').get('.notification__list__item__circle').first().click()
-    cy.get('.notification__header__title').contains('Notifications').should('not.be.visible')
-    cy.get('.notificationButton').click()
+  it('should mark the notification as read after click on the circle', () => {
+    cy.get('.notification__list__item__circle').first().click()
+    cy.get('.notificationButton__notification').should('not.be.visible')
     cy.get('.notification__list__item').first().should('have.class', 'itemRead')
   })
```

Comment:
Why delete this test? Both tests must be true (mark as read when clicking on the notification and mark as read when clicking on the circle)

Reply:
My mistake, since the test was clicking on the circle, I thought it was supposed to mark at read. Which wasn't implemented. So I changed the test.
I'll make two different tests, one for the circle and one for the notification itself.